proto/cosmos/auth/v1beta1/auth.proto

```diff
@@ -18,7 +18,7 @@ message BaseAccount {
   option (cosmos_proto.implements_interface) = "AccountI";
 
   string              address        = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
-  google.protobuf.Any pub_key        = 2 [(gogoproto.jsontag) = "public_key,omitempty"];
+  google.protobuf.Any pub_key        = 2;
```

Comment:
```suggestion
  google.protobuf.Any pub_key        = 2 [json_name = "public_key"];
```

Reply:
is there a reason the field is no public_key?

Reply:
weird part is this shows up as `protobuf:"bytes,2,opt,name=pub_key,json=public_key,proto3" json:"pub_key,omitempty"`

I haven't tested but which json gets picked up?

Reply:
depends on what you use to marshal it, if you use the json package "pub_key" if you use protojson package (which is what we should and eventually must be using) then "public_key".

Reply:
thanks will leave as is for now. We assume it will be used with protojson